src/server/pfs/cmds/cmds.go

```diff
@@ -357,22 +357,30 @@ Files can be read from finished commits with get-file.`,
 		}),
 	}
 
-	var mountPoint string
 	mount := &cobra.Command{
-		Use:   "mount [repo/commit:alias...]",
+		Use:   "mount path/to/mount/point",
 		Short: "Mount pfs locally.",
 		Long:  "Mount pfs locally.",
-		Run: pkgcobra.Run(func(args []string) error {
+		Run: func(c *cobra.Command, args []string) {
 			//lion.SetLevel(lion.LevelDebug)
 			apiClient, err := getAPIClient(address)
 			if err != nil {
-				return err
+				fmt.Fprintf(os.Stderr, "%v\n", err)
+				os.Exit(1)
 			}
 			mounter := fuse.NewMounter(address, apiClient)
-			return mounter.Mount(mountPoint, shard(), parseCommitMounts(args), nil)
-		}),
+			if len(args) != 1 {
+				c.Usage()
+				os.Exit(1)
+			}
```

Comment:
Using that helper results in this output:

```shell
$ pachctl mount
Expected 1 args, got 0
```

Which is a lot less helpful than what it was before by using `c.Usage` : 

```shell
$ pachctl mount
Usage:
  pachctl mount path/to/mount/point [flags]

Flags:
  -n, --block-modulus int   modulus of block shard (default 1)
  -b, --block-shard int     block shard to read
  -m, --file-modulus int    modulus of file shard (default 1)
  -s, --file-shard int      file shard to read
  -h, --help                help for mount
```

The os exit repetition is unfortunate, but thats what the helper is doing under the hood. I'd prefer to leave it as is since `c.Usage` is more descriptive.